src/components/LocalIdentityModal/LocalIdentityModal.js

```diff
@@ -84,13 +85,11 @@ function LocalModal({ address, label, onCancel, onSave }) {
           background: ${theme.surface};
           max-width: calc(100vw - 32px);
 
-          ${breakpoint(
-            'medium',
+          ${above('medium') &&
             `
               /* wide identity badge + paddings */
-              min-width: ${400 + 16 * 2}px;
-            `
-          )};
+              min-width: ${54 * GU}px;
```

Comment:
I think we might've still wanted to leave the calculation there just to be explicit, since we use 32px elsewhere as the padding.